frontend/src/containers/MetricsStep/index.tsx

```diff
@@ -106,12 +138,15 @@ const MetricsStep = () => {
           />
         </MetricSelectionHeader>
       )}
-
       {isShowCrewsAndRealDone && (
         <MetricSelectionWrapper>
           {isLoading && <Loading />}
-          <MetricsSelectionTitle>Board configuration</MetricsSelectionTitle>
-          {isEmpty(errorMessage) ? (
+          <MetricsSelectionTitle>Board configuration </MetricsSelectionTitle>
+
+          {isEmpty(errorMessage) ||
+          (boardInfoFailedStatus != METRICS_DATA_FAIL_STATUS.ALL_FAILED_4XX &&
```

Comment:
please avoid `==` or `!=` in Javascript, unless you are intended to leverage the JS internel type transforming to facilitate some features.

Reply:
replaced with !==